### PR TITLE
[docs] Grammar fixes for intents.rst

### PR DESCRIPTION
**Pull request by Daggy1234**
## Summary

Grammar Fixes

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: docs/intents.rst [hunk 1]
```diff
@@ -202,6 +202,6 @@ On Windows use ``py -3`` instead of ``python3``.
 
 .. warning::
 
-    There is no currently set date in which the old gateway will stop working so it is recommended to update your code instead.
+    There is no current set date on which the old gateway will stop working so it is recommended to update your code instead.
```
Comment (mesub7) [inline docs/intents.rst:205]:
```suggestion
There is no current set date on when the old gateway will stop working so it is recommended to update your code instead.
```

**Review by mesub7 — CHANGES_REQUESTED** [review]

File: docs/intents.rst (hunk 1 above)
Comment (sudosnok) [reply docs/intents.rst:205]: thats not grammatically correct, either `date on which ...` or `date for when ...`

Comment (Daggy1234) [reply docs/intents.rst:205]: Agreed `date on which` sounds the best

**Review by scrazzz — APPROVED** [review]

File: docs/intents.rst (hunk 1 above)
Comment (mesub7) [reply docs/intents.rst:205]: Fair enough.

**Review by mesub7 — APPROVED** [review]

**Review by ImVaskel — APPROVED** [review]

File: docs/intents.rst [hunk 2]
```diff
@@ -5,7 +5,7 @@
 A Primer to Gateway Intents
 =============================
 
-In version 1.5 comes the introduction of :class:`Intents`. This is a radical change in how bots are written. An intent basically allows a bot to subscribe into specific buckets of events. The events that correspond to each intent is documented in the individual attribute of the :class:`Intents` documentation.
+In version 1.5 comes the introduction of :class:`Intents`. This is a radical change in how bots are written. An intent basically allows a bot to subscribe to specific buckets of events. The events that correspond to each intent is documented in the individual attribute of the :class:`Intents` documentation.
```
Comment (Rapptz) [inline docs/intents.rst:8]: The previous sentence wasn't wrong but this change is acceptable I guess.

File: docs/intents.rst [hunk 3]
```diff
@@ -121,7 +121,7 @@ It should be noted that certain things do not need a member cache since Discord
 - :func:`on_reaction_add` will have the ``user`` parameter be a member when in a guild even if cache is disabled.
 - :func:`on_raw_reaction_add` will have :attr:`RawReactionActionEvent.member` be a member when in a guild even if cache is disabled.
 - The reaction add events do not contain additional information when in direct messages. This is a Discord limitation.
-- The reaction removal events do not have the member information. This is a Discord limitation.
+- The reaction removal events do not have member information. This is a Discord limitation.
```
Comment (Rapptz) [inline docs/intents.rst:124]: Again, previous sentence wasn't wrong but this change is acceptable.

File: docs/intents.rst (hunk 1 above)
Comment (Rapptz) [inline docs/intents.rst:205]: This change is wrong.

Anyway it might be better to remove this section entirely since it no longer works.

**Review by Rapptz — CHANGES_REQUESTED** [review]

**Review by Rapptz — CHANGES_REQUESTED** [review]
The section needs to be removed not just the warning.

**Comment by Daggy1234** [comment]
Removed the section, but I did keep the final line

**Review by Rapptz — APPROVED** [review]

**Pull request merged**